### DISCUSSION TITLE
Open pdf via Citation Key

### DISCUSSION THREAD

**Discussion by xracoonx** [retorquere/zotero-better-bibtex#2619] [Q&A]
Is it possible to open a pdf that is attached in Zotero from the command line via the Citation Key?

I read about the possibility to open pdfs in Zotero via 

`zotero://open-pdf/library/items/[Zotero-ID]`

Unfortunately, all I have on my end is the Better BiBbTeX generated citation key.

Any help much appreciated.

**Comment by retorquere** [comment]
This needs a [change in Zotero](https://forums.zotero.org/discussion/comment/411727/#Comment_411727) before it can work.

**Comment by xracoonx** [comment]
Thanks. Just to be certain that I understand this. They say that "only regular items have citekeys". I don't care about a specific pdf of a "regular item". But I take it that it is also not possible to open the first attachment (or so) of a regular item.

**Reply by retorquere** [reply]
The they you are referring to is me, but that's not the problem - the problem is that there is one place where I can inject my "translate citation key to Zotero item", and that is used in two places (once for "open this item", and once for "open the attachment on this item"), and I cannot distinguish between the two. Until zotero makes it so I can see this distinction there is nothing I can do.

**Comment by xracoonx** [comment]
I might still not be following. But is it possible then to show the item in Zotero by only having the citation key? (Of course opening the attachment would be ideal, but showing the item in Zotero would be great too.)

**Comment by retorquere** [comment]
That is possible using `zotero://select/items/@<your citekey>`.

**Reply by xracoonx** [reply]
That works well, thanks.
